.kitchen.yml

```diff
@@ -33,6 +33,18 @@ platforms:
       run_command: /sbin/init
       volume:
         - /sys/fs/cgroup:/sys/fs/cgroup:ro
+  - name: centos-6
+    driver_config:
+      image: stackstorm/packagingtest:centos6
+      platform: rhel
+      run_command: /sbin/init
+  - name: centos-7
+    driver_config:
+      image: stackstorm/packagingtest:centos7
+      platform: rhel
```

Comment:
I double-checked, you were right.
There is `centos` option according to https://github.com/test-kitchen/kitchen-docker#platform